Whiskey/en-US/about_Whiskey_DotNet_Task.help.txt

```diff
@@ -0,0 +1,136 @@
+TOPIC
+    about_Whiskey_DotNet_Task
+
+SUMMARY
+    Runs dotnet.exe commands.
+
+DESCRIPTION
+    The `DotNet` tasks runs dotnet.exe commands. .NET Core is installed for you into a ".dotnet" directory (in the directory of your whiskey.yml file). You can control what version of the .NET Core SDK to download and use with the "SdkVersion" parameter.
+    
+    If a command accepts any paths to files, pass them to the `Path` property. Wildcards are allowed and the `DotNet` task will run the dotnet.exe command once for each path.
+
+    Pass any command arguments with the `Argument` property.
+
+    The task writes a detailed build log to the build output directory. If the `Path` property has no value, the file is named `dotnet.COMMAND.log`. Otherwise, there will be a log file for each path named `dotnet.build.FILE_NAME.log`, where `FILE_NAME` is the name poertion of the path.

```

Comment:
`dotnet.build.FILE_NAME.log` should be `dotnet.COMMAND.FILE_NAME.log`

Also typo on `poertion`

Reply:
Still needs fixing.